### DISCUSSION TITLE
Welcome to AirfoilEditor Discussions!

### DISCUSSION THREAD

**Discussion by jxjo** [jxjo/AirfoilEditor#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by goney3** [comment]
Hello Jxjo,
Congratulations on the release of Airfoil Editor 4.0! I'm really enjoying the new version.
I've found it to be very intuitive and easy to use, even without reading the documentation. As a hobbyist, I'm thrilled to have software like this to experiment with. I'm running it on Windows 11, and it works perfectly. The on-the-fly polars are a fantastic feature!
I do have a question about comparing the stall conditions of different airfoils (e.g., a sharp drop-off vs. a gentle curve). I've been learning about gyroplanes, which often use the NACA 8-H-12 airfoil. It's known for its forgiving stall characteristics, and I was wondering if there's a setting in Xoptfoil2 to analyze or optimize for "gentle stalls"?
I also have a couple of minor suggestions for the user interface. In the "Optimize" section, the edit boxes for a graph only appear when that specific graph is selected. It might be more convenient if the drop-down menus automatically changed to the relevant graph type for easier tweaking. Additionally, is there an option for a graph of Alpha vs. the Center of Pressure (XCp)?
Thank you again for your excellent work on this software.

**Reply by jxjo** [reply]
Hello Keith,

... I'm happy, that you like the new AirfoilEditor!  And thank you for the flowers ... ;-) 

Regarding stall characteristics: Mmh - I don't know, if it makes too much sense to optimze stall based on xfoil which has its limitations when alpha gets close to stall... In optimizations I typically have an extra operating point to control the high lift properties of the airfoil.
This operating point is based on alpha (let's say 12°) and has a cl target value (let's say 1.35). In the concert with the other operating points, this point influences very much the leading edge curvature which mainly influences high lift ...

Thank you for the suggestions! But sorry, which "edit boxes" do you mean? Maybe you could send a screen shot?  

xcp is not available. Couldn't be cm used to assess movement of xcp?

Best regards

Jochen

**Reply by goney3** [reply]
Thank you Jochen!
Ah, my bad - I meant "Op Point Definitions" squares = "edit boxes".
Cm can definitely be for xcp, I just find it a little harder to "see" and compare is all (personal preference). I guess I've just been using XFLR5 for too long hehe
<img width="1317" height="923" alt="image" src="https://github.com/user-attachments/assets/c22215e3-333f-43f2-a09e-1de0647201f1" />

The same airfoils at the same Reynolds numbers gives me this in Airfoil Editor:
<img width="1650" height="788" alt="image" src="https://github.com/user-attachments/assets/0f98b6a4-6932-49d0-ad18-b7df5cc8a4f4" />
Where the line stops in Cl/alpha diagram is where stall occurs, I assume? :)

**Reply by jxjo** [reply]
Hello Keith,

... ah - I see. I call the little boxes "mouse helper" ;-) If an operating point definition is editable in a certain diagram depends, if the variables of the diagram axes match the specification variable of the operating point (either alpha or cl) and the optimization variable e.g. cd - where cd and cl/cd are calculated if possible. 
In the new version 4.1 the logic to show and edit an operating point definition is refactored. So hopefully it's better now ;-) 
If you still find a strange behaviour, please give me a feedback.

--

Yep, for the polar calculation a feature called "auto range" is used. In this case no alpha_min alpha_max is needed to define the range of the polar. Instead the pola points are calculated unitl 2 points behind cl_max or cl_min. This helps very much that the polar shows the interesting part before the stall (otherwise the polar is stretched to the highest cd - quite often such polars are presented in screenshots. It looks impressive - but you don't see anything ;-))

In the case of the NACA 8 H-2, it would be better to switch off "auto range" which is currently not possible. 
But again, the question is, if Xfoil shows realisitc value behind cl_max...

Best regards!
